data/effects/swirl.effect

```diff
@@ -0,0 +1,144 @@
+#include "shared.effect"
```

Comment:
- Missing License and Copyright header.
- Wrong include file.

Reply:
- Include file is now `"../base.effect"`.
- Added License and Copyright header.